src/tests/counts.test.js

```diff
@@ -0,0 +1,20 @@
+import countMeals from '../__mocks__/itemCount.js';
```

Comment:
**Changes Required**
- [ ] There seem to be an issue importing your `canvas.node` class which is making the test fail. Let us try to fix this. Below is the error I get when I run `npm test` on the `development` branch of your project.
![failed](https://user-images.githubusercontent.com/7220167/132782723-e8663fbd-dc86-4872-ba49-0ae024fe9ba1.png)

Reply:
We cloned a new repo and immediately after npm install the tests we're working fine. We're unable to find any error.